Tasks/codepush-release-cordova/codepush-release-cordova.js

```diff
@@ -83,13 +81,21 @@ function performDeployTask(accessKey, appName, appStoreVersion, platform, deploy
         console.error("Access key required");
         tl.setResult(1, "Access key required");
     }
-  
+
     // Ensure all other users are logged out.
     ensureLoggedOut();
-  
+
     // Log in to the CodePush CLI.
     executeCommandAndHandleResult("login", /*positionArgs*/ null, { accessKey: accessKey });
 
+    // Try to find cordova and prepare the environment if not found. 
+    if (!which("cordova")) {
+        console.log("cordova cli not found. Installing...");
+        exec("npm install cordova");
+        var originalPath = process.env["PATH"];
+        process.env["PATH"] = path.join(process.cwd(), "node_modules", ".bin") + (process.platform == "win32" ? ";" : ":") + originalPath;
```

Comment:
That's a good point. I'll move it outside the which command so that if we have cordova installed locally, we default to using that. Initially, i didn't want to modify the path until afterwards so we could use globally installed cordova even if it was installed locally, but I think it makes more sense to default to the local one.